migrations/2_deploy_contracts.js

```diff
@@ -2,22 +2,42 @@ var STKToken = artifacts.require("./STKToken.sol");
 var STKChannel = artifacts.require("./STKChannel.sol");
 var SafeMathLib = artifacts.require("./SafeMathLib.sol");
 var STKChannelLibrary = artifacts.require('./STKChannelLibrary.sol');
-module.exports = function(deployer)
+module.exports = function(deployer, network, accounts)
 {
-  deployer.deploy(SafeMathLib).then(function()
-  {
-    return deployer.link(SafeMathLib,STKChannelLibrary).then(function()
+
+  if(network === "development") {
```

Comment:
This seems repetitive (DRY -> Don't Repeat Yourself). Can you collapse these into one?